packages/tools/devtools/devtools-core/src/messaging/devtools-messages/ContainerList.ts

```diff
@@ -29,12 +31,17 @@ export namespace ContainerList {
 		 * List of keys for the Containers registered with the Devtools.
 		 */
 		containers: ContainerKey[];
+
+		/**
+		 * List of keys for the Container Runtimes registered with the Devtools.
+		 */
+		containerRuntimes?: ContainerKey[];
```

Comment:
Does this need to be a separate list? I think the only pieces of info the view needs is
1. The unique key
2. Whether or not the associated "container" supports state modification.

So, I think we can just have a single "containers" list?

Reply:
I kept it separately thinking we'd have to render `Container` and `ContainerRuntime` separately in the devtools pane, just as we did for `DataObjects`. 

Made this decision since there still is a subtle difference in two concepts. If you think we should just render the devtools instances as `Containers`, no problem making this into a single list.

Reply:
I think the difference in this case is less meaningful to the end user, so having a single combined view probably makes the most sense IMO